common/src/types/v0/transport/replica.rs

```diff
@@ -31,8 +129,8 @@ pub struct Replica {
     pub name: ReplicaName,
     /// uuid of the replica
     pub uuid: ReplicaId,
-    /// id of the pool
-    pub pool: PoolId,
+    /// pool on which the replica resides
+    pub pool: ReplicaPool,
```

Comment:
`PoolRef` ?

Reply:
`PoolRef` used inside ReplicaSpec for now. As a  modification if we feel to change these as well to PoolRef, will take up as a refactor task.